pkg/server/server.go

```diff
@@ -1004,6 +982,15 @@ func (s *Server) startPersistingHLCUpperBound(
 	return nil
 }
 
+// collectCallCounts is a middleware to collect telemetry on the call counts
+// of paths served by `inner`.
+func collectCallCounts(inner http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		telemetry.Count(fmt.Sprintf("HTTP: %s -> %s", req.Method, req.URL.EscapedPath()))
```

Comment:
Can we follow the pattern of other calls to `telemetry.Count` and do something like `http.%s.%s`?

Reply:
Done